contracts/base/Settings.sol

```diff
@@ -385,6 +397,28 @@ contract Settings is Pausable, SettingsInterface {
         );
     }
 
+    /**
+        @notice Sets the collateral buffer value.
+        @param newCollateralBuffer the new collateral buffer value.
+     */
+    function setCollateralBuffer(uint256 newCollateralBuffer)
+        external
+        onlyPauser()
+        whenNotPaused()
+    {
+        require(collateralBuffer != newCollateralBuffer, "NEW_VALUE_REQUIRED");
+        require(newCollateralBuffer > 0, "MUST_PROVIDE_COLLATERAL_BUFFER");
```

Comment:
probably mentioning the collateral value must be a positive number would better clarify the intention of this validation:

MUST_PROVIDE_COLLATERAL_BUFFER  => COLLATERAL_MUST_BE_POSITIVE

This is just a comment.

Reply:
The Settings contract will be refactored in the next PR. Now it will contain a mapping for the settings